de.manumaticx.pagingcontrol/controllers/widget.js

```diff
@@ -7,7 +7,13 @@ _.defaults(args, {
     tabs: false,
     scrollOffset: 40,
     height: args.tabs ? 48 : 5,
-    width: Ti.UI.FILL
+    width: Ti.UI.FILL,
+    findScrollableView: true
+});
+
+// xml boolean args is string ("false" == true) 
+_.each(['tabs', 'findScrollableView'], function(key){
+    args[key] = JSON.parse(args[key]);    
```

Comment:
ok, can we surround it with try-catch then, and only apply if it's a valid boolean (or boolean string)?